.gitignore

```diff
@@ -15,3 +15,4 @@
 *.wasm
 
 test_data
+tmp
```

Comment:
?

Reply:
ignore all tmp directories... I prefer to use `--datadir tmp/alice` when testing, would it be ok if we added this to the gitignore?

Reply:
why not, if that helps you :)